src/attr/_make.py

```diff
@@ -1924,6 +1925,45 @@ def _assign_with_converter(attr_name, value_var, has_on_setattr):
     )
 
 
+if PY2:
+
+    def _unpack_kw_only_py2(attr_name, default=None):
+        """
+        Unpack *attr_name* from _kw_only dict.
+        """
+        if default is not None:
+            arg_default = ", %s" % default
+        else:
+            arg_default = ""
+        return "%s = _kw_only.pop('%s'%s)" % (
+            attr_name,
+            attr_name,
+            arg_default,
+        )
+
+    def _unpack_kw_only_lines_py2(kw_only_args):
+        """
+        Unpack all *kw_only_args* from _kw_only dict and handle errors.
+        """
+        assert kw_only_args
+        lines = ["try:"]
+        lines.extend(
+            "    " + _unpack_kw_only_py2(*arg.split("="))
+            for arg in kw_only_args
+        )
+        lines += """\
+except KeyError as _key_error:
+    _msg = '__init__() missing required keyword-only argument: %s'
```

Comment:
I don't think creating a variable (`_msg`) is necessary here?

Reply:
I had problems with linter here, as line was too long.
Put blacked code in triple-quotes.